scripts/en-leaks-setup-mocks.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
looks like this script is for generating PL but not mocks?

Reply:
You are right. I can change the script name

Reply:
The only reason I added a script is that we can log an error saying `Failed to generate pseudo-loc. This may result in some unexpected test failures`

Reply:
rename to `detect-english-leaks-setup.sh` if you prefer to keep it.?